data/json/recipes/recipe_vehicle.json

```diff
@@ -1,4 +1,32 @@
 [
+  {
+    "result": "rebar_plate",
```

Comment:
Is this right?

Reply:
Nope, it should be the makeshift steam engine. Thanks for the review.

Reply:
Fixed.